### DISCUSSION TITLE
Write HR value to text file

### DISCUSSION THREAD

**Discussion by SouljaVR** [200Tigersbloxed/HRtoVRChat_OSC#10] [Ideas]
Would it be possible to have HR values written to a text file? Similar to how HRToVRC can read from a text file, but the other way around lol. I want to be able to display my heart rate on other platforms while streaming HR data to my avatar. Having it written live to a text file would make this really easy for me since I cant read values from the game once HRtoVRC takes control of the port. Thank you :)

I would just want the HR values written in plain integer form into the text file, nothing more so its easy to parse. Thank you again.

**Comment by 200Tigersbloxed** [comment]
You can use the AppBidge and InitClient to receive updates from the server, you just have to call InitClient once HRtoVRChat_OSC is launched.

**Reply by SouljaVR** [reply]
Okay thank you!
